db/migrate/20180208165026_create_books.rb

```diff
@@ -3,9 +3,9 @@ def change
     create_table :books do |t|
       t.string :name
       t.string :author
-      t.string :sDescription
-      t.string :lDescription
-      t.string :image
+      t.string :short_description
+      t.string :long_description
```

Comment:
better but should be more useful, summary and description we talked would be a better name?